### ISSUE TITLE
rotation problem on asus transformer tf101

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#210]
```
What steps will reproduce the problem?
1.open ebookdroid
2.rotate the tablet
3.

What is the expected output?
The bookshelf rorated

What do you see instead?
A. Blank screen

What version of the product are you using?
Last version as of 20  april 2012

On what operating system
Android 4.0.3


What device are you using?


Please provide any additional information below.

```

Original issue reported on code.google.com by `f.scarr...@gmail.com` on 20 Apr 2012 at 10:23

**Comment by GoogleCodeExporter** [comment]
```
Version 1.4.0 worked correctly version 1.4.1 introduced the problem
```

Original comment by `f.scarr...@gmail.com` on 21 Apr 2012 at 6:02

**Comment by GoogleCodeExporter** [comment]
```
Problem is not present on galaxy s with android Gingerbread
```

Original comment by `f.scarr...@gmail.com` on 21 Apr 2012 at 9:42

**Comment by GoogleCodeExporter** [comment]
```
This problem appears if Android recreate activity on rotation.

```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 21 Apr 2012 at 10:48
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fix both in 1.4.x/ and trunk/
Pls try the following:
http://code.google.com/p/ebookdroid/downloads/detail?name=ebookdroid-1.4.2-dev-r
1217.apk&can=2&q=#makechanges
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 21 Apr 2012 at 12:06
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Ok, I tried, it works now, thank you
```

Original comment by `f.scarr...@gmail.com` on 21 Apr 2012 at 5:09
